tools/clang/unittests/HLSL/PixDiaTest.cpp

```diff
@@ -2920,4 +2923,57 @@ float4 DeeperInlinedFunction(in BuiltInTriangleIntersectionAttributes attr, int
                        ColorRegisterNumberWhenCalledFromOuterForDeeper);
 }
 
-#endif
\ No newline at end of file
+TEST_F(PixDiaTest,
+       DxcPixDxilDebugInfo_InlinedFunctions_CalledTwiceInSameCaller) {
+  if (m_ver.SkipDxilVersion(1, 2))
```

Comment:
You compile to `lib_6_6` in this code, so I think the minimum required DXIL & validator should be `(1, 6)`, not `(1, 2)`.

Reply:
Ah, copy-pasta. Thanks.